src/backscratcher/StrategyProxy.sol

```diff
@@ -283,13 +290,14 @@ contract StrategyProxy {
 
     function harvest(address _gauge, address[] calldata _tokens) external {
         require(strategies[_gauge] == msg.sender, "!strategy");
+
         uint256[] memory _balances = new uint256[](_tokens.length);
 
         for (uint256 i = 0; i < _tokens.length; i++) {
             _balances[i] = IERC20(_tokens[i]).balanceOf(address(proxy));
         }
 
-        proxy.safeExecute(_gauge, 0, abi.encodeWithSignature("getReward()"));
+        proxy.safeExecute(_gauge, 0, claimRewardsString[_gauge]);
```

Comment:
Would it be helpful to have a default/fallback like so?
```
proxy.safeExecute(_gauge, 0, 
claimRewardsString[_gauge].length == 0 ? abi.encodeWithSignature("getReward()") : claimRewardsString[_gauge]);
```

Please ignore if you anticipate that `claimRewardsString` can always be reliably defined when `approveStrategy` is invoked.

Reply:
We cannot pass "" into the calldata parameter, we could try passing "0x0" but I am confident we can encode the ABI for the string we need.